addons/sale/models/sale_order.py

```diff
@@ -1049,6 +1049,10 @@ def _notify_get_groups(self, msg_vals=None):
                 if group_name not in ('customer', 'portal'):
                     group_data['has_button_access'] = True
 
+                if group_name in ('customer'):
```

Comment:
Note that `('customer') == 'customer'`, so `'custom' in ('customer') == True`
```suggestion
                if group_name in ('customer',):
```

or 
```suggestion
                if group_name == 'customer':
```

Reply:
Uhh, nice hint, so it is better to use explicit lists I guess or a tuple as you pointed out 🙏🏼

Reply:
@william-andre 
Thx, and fixed...such a role model you are...I wish more developers would act like you on Github and in general 🥇